package.json

```diff
@@ -19,23 +19,23 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.6.4",
-    "@babel/core": "^7.6.4",
+    "@babel/core": "^7.11.0",
```

Comment:
I changed the version of babel/core because I was trying to get rid of the following error message during Travis checks with PHP: 7.4 and TEST_SUITE=integration. 
```
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.

npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

npm WARN deprecated core-js@1.2.7: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.

npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated

npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/chokidar/node_modules/fsevents):

npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ETARGET

npm ERR! notarget No matching version found for @babel/helper-module-transforms@^7.11.0

npm ERR! notarget In most cases you or one of your dependencies are requesting

npm ERR! notarget a package version that doesn't exist.

npm ERR! notarget 

npm ERR! notarget It was specified as a dependency of '@babel/core'

npm ERR! notarget 

npm ERR! A complete log of this run can be found in:

npm ERR!     /home/travis/.npm/_logs/2020-07-30T21_27_39_641Z-debug.log

Makefile:14: recipe for target 'javascript' failed

make: *** [javascript] Error 1

The command "make dev" failed and exited with 2 during .

```

Reply:
@kongtiaowang I tried changing babel/core version because I had the error message above during Travis checks with PHP: 7.4 and TEST_SUITE=integration. The problem did not show up with php7.3. Is this solution correct or there is another solution I am not seeing?